### PR TITLE
cp-282:

### PR DESCRIPTION
**Pull request by carrbrpoa**
- Contact Us now sends email to channelprogram address;
- Mark Contact fields as mandatory.

### CODE REVIEW

File: app/views/pages/api/contacts/post.liquid [hunk 1]
```diff
@@ -8,15 +8,35 @@ response_headers: >
...
+    assign dataToEmail = '{}' | parse_json
+    assign dataToEmail = dataToEmail | add_hash_key: "email", context.params.email
+    assign dataToEmail = dataToEmail | add_hash_key: "name", context.params.name
+    assign dataToEmail = dataToEmail | add_hash_key: "company", context.params.company
+    assign dataToEmail = dataToEmail | add_hash_key: "company_type", context.params.company_type
+    assign dataToEmail = dataToEmail | add_hash_key: "message", context.params.message
+    graphql notifyEmail = 'emails/contact_us', data: dataToEmail
+    log notifyEmail, type: 'debug'
+  elsif result_create.errors.size > 0
+    assign errorMessagesFromDB = result_create.errors | map: 'message'
```
Comment (erica-channel-program) [inline app/views/pages/api/contacts/post.liquid:31]: can we log an error here?

File: app/views/pages/api/contacts/post.liquid [hunk 2]
```diff
@@ -8,15 +8,35 @@ response_headers: >
...
+    assign messages = messages | concat: errorMessagesFromDB
+    assign return = return | add_hash_key: "messages", messages
+  else
+    assign message = "There is a problem registering your contact right now, please try again later"
+    assign messages = messages | add_to_array: message
```
Comment (erica-channel-program) [inline app/views/pages/api/contacts/post.liquid:35]: can we log an error here?

File: app/views/pages/api/contacts/post.liquid (hunk 1 above)
Comment (carrbrpoa) [reply app/views/pages/api/contacts/post.liquid:31]: Sure, will do it in next commit

File: app/views/pages/api/contacts/post.liquid (hunk 2 above)
Comment (carrbrpoa) [reply app/views/pages/api/contacts/post.liquid:35]: Sure, will do it in next commit

Comment (carrbrpoa) [reply app/views/pages/api/contacts/post.liquid:35]: Data that caused the `else` case is being logged in line 11, but we can log `messages` by the end

**Review by erica-channel-program — APPROVED** [review]

**Pull request merged**